build.gradle

```diff
@@ -39,8 +65,10 @@ subprojects {
 
     // Set up java codestyle checking and correction.
     plugins.withId("java") {
-        plugins.apply('checkstyle')
```

Comment:
Was this removal intentional?

Reply:
yes.  When checkstyle is ACTUALLY run (on release) it fails.   ANy style checks NOT part of spotless should be moved there, we don't need more than one checkstyle (when they do different things)